features/staging_commands.feature

```diff
@@ -93,13 +93,19 @@ Feature: Enable command behaviour when attached to an UA staging subscription
 
     @series.lts
     @uses.config.machine_type.lxd.container
-    Scenario Outline: Attached enable esm-ros on a machine
+    Scenario Outline: Attached enable ros on a machine
         Given a `<release>` machine with ubuntu-advantage-tools installed
         When I attach `contract_token_staging` with sudo
         And I run `ua status --all` as non-root
         Then stdout matches regexp
         """
-        esm-ros       yes                enabled            ROS Extended Security Maintenance \(ESM\)
+        ros           yes                disabled           Robot Operating System security fixes
+        """
+        When I run `ua enable ros --assume-yes` with sudo
```

Comment:
Don't we need the `--beta` flag here ?

Reply:
Also, another check we can add is to verify that ros is only delivering the `security` apt file while `ros-updates` is delivering both updates and security apt files

Reply:
yes good catch on the `--beta`! I haven't run this yet because the staging server hasn't been updated yet.

And yes I should add an explicit check for which of the two sources is available for each entitlement.